tests/test_dht_node.py

```diff
@@ -29,6 +30,11 @@ def run_protocol_listener(port: int, dhtid: DHTID, started: mp.synchronize.Event
     print(f"Finished peer id={protocol.node_id} port={port}", flush=True)
 
 
+# note: we run grpc-related tests in a separate process to re-initialize all global states from scratch
```

Comment:
All changes in this file are strictly the following:
* remove boilerplate code that would sparn processes inside tests
* mark tests as forked and/or asyncio where necessary